packages/itwinui-react/src/core/InputWithDecorations/InputWithDecorations.tsx

```diff
@@ -60,33 +65,39 @@ const InputWithDecorationsButton = React.forwardRef((props, ref) => {
   );
 
   return (
-    <IconButton
+    <InputFlexContainerButton
       ref={ref}
       size={size ?? contextSize}
-      styleType='borderless'
       disabled={localDisabled ?? isDisabled}
       {...rest}
     >
       {children}
-    </IconButton>
+    </InputFlexContainerButton>
   );
 }) as PolymorphicForwardRefComponent<
   'span',
-  React.ComponentProps<typeof IconButton>
+  React.ComponentProps<typeof InputFlexContainerButton>
+>;
+
+const InputWithDecorationsIcon = React.forwardRef((props, ref) => {
+  return <InputFlexContainerIcon ref={ref} {...props} />;
+}) as PolymorphicForwardRefComponent<
+  'span',
+  React.ComponentProps<typeof InputFlexContainerIcon>
```

Comment:
Why not just reassign it?

```jsx
const InputWithDecorationsIcon = InputFlexContainerIcon;
```

Reply:
Also add separators and `displayName`

Reply:
Why did the length of other separators change?

![image](https://github.com/iTwin/iTwinUI/assets/9084735/269bdf91-88c7-47b3-a7db-2440b73a092e)

Maybe the vscode snippet is the wrong length and should be fixed?

Reply:
I assumed the VSCode snippets to be the correct lengths. That's why I changed the other existing separator comments to match the VSCode snippet's length.

Upon doing a global search, looks like the VSCode separator length is longer than the separators we're using in our repo. I added a bullet point to the after PR TODOs to fix this.

For now, I manually updated the separators to use the existing separator length instead of the length from the VSCode snippet (https://github.com/iTwin/iTwinUI/pull/1881/commits/1d5897fff1208b0706d36ef9e6a1ddd5b5068465).

Reply:
Feel free to also update the separate length in this PR. It's too minor to warrant a new PR 😅

Reply:
Done 🙂 ([`30c62de` (#1881)](https://github.com/iTwin/iTwinUI/pull/1881/commits/30c62de52fec366b97eaecb0ee2bec2d4b409fed))